### PR TITLE
docs: add recruiting section

### PR DESCRIPTION
**Pull request by tancnle**
# Description

This PR moves recruitment content from marketing to the people operations section.

## Type of change

- [x] Documentation (update or new)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: docs/peopleops/recruiting/index.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Recruiting
+
+- [Pitch](pitch.md)
+- [Recruitment Ad](recruitment-ad.md)
```
Comment (tancnle) [inline docs/peopleops/recruiting/index.md:4]: **thought:** These two docs are very similar. Should we drop one of them? 🤔

File: docs/peopleops/recruiting/pitch.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ pipeline
 
 ## Current Tech Stack
 
-Ruby on Rails, Rabbit MQ, Docker, Angular, C/C++, TypeScript, Bootstrap, CoffeeScript, Python, Cpp.
```
Comment (tancnle) [inline docs/peopleops/recruiting/pitch.md:30]: **thought:** CPP is C++ so we do not need to repeat.

File: docs/peopleops/recruiting/index.md (hunk 1 above)
Comment (ghost) [reply docs/peopleops/recruiting/index.md:4]: Yes, probably should merge them into one marketing file with both ad and pitch

**Review by ghost — APPROVED** [review]
Great idea! Good to see handover is progressing

File: docs/peopleops/recruiting/index.md (hunk 1 above)
Comment (tancnle) [reply docs/peopleops/recruiting/index.md:4]: Coolio. I will merge them in a follow-up PR.

**Pull request merged**